libraries/Microsoft.Bot.Builder.Ai/LocaleConverterMiddleware.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.Bot.Builder.Middleware;
+using Microsoft.Bot.Schema;
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder.Ai
+{
+    /// <summary>
+    /// Middleware to convert messages between different locales specified
+    /// </summary>
+    public class LocaleConverterMiddleware : IReceiveActivity
+    {
+        private LocaleConverter localeConverter;
+        private string fromLocale;
+        private string toLocale;
+
+        public LocaleConverterMiddleware(string fromLocale, string toLocale)
```

Comment:
>string fromLocale [](start = 41, length = 17)

so if I wanted to convert from 4 different languages to my bots language I would need to add 4 separate pieces of middleware? That seems odd.

Reply:
I have updated the LocaleConverterMiddleware to take a delegate function implemented by the developper to switch the from Locale